plugins/modules/ec2_instance.py

```diff
@@ -1699,6 +1751,48 @@ def handle_existing(existing_matches, state):
     return result
 
 
+def enforce_count(existing_matches, module):
+    exact_count = module.params.get('exact_count')
+
+    try:
+        current_count = len(existing_matches)
+        if current_count == exact_count:
+            module.exit_json(
+                changed=False,
+                msg='{0} instances already running, nothing to do.'.format(exact_count)
+            )
+
+        elif current_count < exact_count:
+            to_launch = exact_count - current_count
+            module.params['to_launch'] = to_launch
+            # launch instances
+            try:
+                ensure_present(existing_matches=existing_matches, desired_module_state='present')
```

Comment:
This would preclude folks from waiting on running/ok and feels a little off.

Reply:
Modified to enable use of ```wait``` and ```state```.